src/runtime/methodbinder.cs

```diff
@@ -519,6 +541,13 @@ static IntPtr HandleParamsArray(IntPtr args, int arrayStart, int pyArgCount, out
                         op = Runtime.PyTuple_GetItem(args, paramIndex);
                     }
                 }
+                if (isOperator && paramIndex == 0)
+                {
+                    // After we've obtained the first argument from Python, we need to skip the first argument of the CLR
+                    // because operator method is a bound method in Python
+                    paramIndex++; // Leave the first .NET param as null (margs).
+                    parameter = pi[paramIndex];
+                }
```

Comment:
It makes more sense to put this block before the assignment to parameter, so that it doesn't need to be done again.

Reply:
The cleanest approach would be:
```
int startingParamIndex = isOperator ? 1 : 0;
for(int paramIndex = startingParamIndex; ...)
```

Reply:
Actually we would still need to do `op = Runtime.PyTuple_GetItem(args, paramIndex);` in a previous line so we can't completely skip `paramIndex=0` https://github.com/pythonnet/pythonnet/blob/1f40564cbff0f109b6042973527b032be194b7e8/src/runtime/methodbinder.cs#L519

I tried to convey that in the comment here, "After we've obtained the first argument from Python"

Reply:
I see, so we want op to refer to args[0], while parameter refers to pi[1].  If there were a third argument, we would want op to refer to args[1], while parameter refers to pi[2].  So why does the 'if' condition include 'paramIndex == 0'?

Reply:
Since we're looking at binary operators here there would be 1 python arg (just `b` from `a.__add__(b)`), while the CLR operator e.g. op_Addition has 2 args. so we need to take the first arg `b` from Python and translate it into the second arg `b` of the CLR operator

I wrote it in this comment in another function called before this one, but maybe it's worth writing the above here too?
https://github.com/pythonnet/pythonnet/pull/1324/files#diff-237dc718ad5424f5476d2ebbbc4e5164c95901bd2a4c2daaf8ca6fc9f9d3a5fbR701-R702

Reply:
Yes, and the paramIndex==0 condition should be omitted if it is not needed.

Reply:
This will be removed entirely anyway in the next PR since `isOperator` is no longer passed into `TryConvertArguments` as discussed https://github.com/pythonnet/pythonnet/pull/1324/files